src/UI/system-bottom-bar.tsx

```diff
@@ -3,15 +3,21 @@ import { System } from '../utils/system-generator/generate-sector';
 interface systemInformation {
     playerSystem: System;
 }
-
-export const SystemBottomBar = ({ playerSystem }: systemInformation) => {
+interface toggles {
+    toggleResources: boolean;
+    toggleBuildings: boolean;
+    setToggleResources: any;
```

Comment:
You could probably use `() => void` here instead of `any` for the setters. That way, at least it's enforcing any argumentless function rather than any value. Leaving anys in your interface can cause havoc later down the line and make it hard to debug issues (undefined is considered part of any and is usually the culprit for a type related bug). Sometimes though, if the amount of time it will take to figure out how to type it is going to be a complete blocker, it's better to leave it as any, note it, and then change it later (concept of "tech debt").

Reply:
Got it. I used this as a temporary work around. It was throwing an error when I used Function

Reply:
It looks like () => void is throwing an error. I'll mark it for later if it  becomes an issue